vendor/maplibre-override.css

```
@@ -0,0 +1,31 @@
+.maplibregl-ctrl-top-right .maplibregl-ctrl {
+  margin: 60px 25px 0 0;
```

Once you increase the size of the buttons to 40px, you need to update the margins:
```suggestion
  margin: 72px 24px 0 0;
```